docs/content/manual/rancher-integration/lh-hardend-rancher.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Longhorn in an hardened cluster
+---
+
+**Given** Hardened Downstream (RKE2/RKE1/K3s) cluster in Rancher with CIS 2.6
```

Comment:
In the case of Rancher 2.6.9 and 2.7.1, the CIS benchmark version used may differ, with Rancher 2.6.9 using CIS benchmark version 2.1.0/2.1.1 and Rancher 2.7.1 using CIS benchmark version 3.0.0.
Do we need to use a specific CIS benchmark version for testing?

![Screenshot_20230404_200441](https://user-images.githubusercontent.com/104337648/229788423-c988ca23-f1ec-453d-a124-094f7cedfe01.png)
![Screenshot_20230404_194945](https://user-images.githubusercontent.com/104337648/229788653-8f40384a-8877-4133-9ab1-7eb483bb3e7d.png)

Reply:
This app in the attached screenshot is for scanning.

As per https://ranchermanager.docs.rancher.com/v2.6/pages-for-subheaders/rancher-v2.6-hardening-guides CIS version is 1.6 for Rancher v2.6.x.
For v2.7 its CIS 1.6 and 1.20 & v1.23 as per https://ranchermanager.docs.rancher.com/pages-for-subheaders/rancher-v2.7-hardening-guides
I'll update the version. Thanks